iroh-net/src/magicsock.rs

```diff
@@ -1762,25 +1762,30 @@ impl Actor {
         };
 
         loop {
+            inc!(Metrics, actor_tick_main);
```

Comment:
all these `_tick_` metrics are basically trace-level metrics.  is this really a good idea?  We have trace-level logging for this.  What do you expect these to be used for?

Reply:
we used these quite a lot in the past to detect the health of tokio loops

Reply:
So 2 reasons for this:
- In general I'd like to move away from trace logging and shift towards trace metrics. Much easier to reason with it on a chart. Though logs do have their place where explicit order is really important.
- We actually used these in development MUCH more and we're more useful to us than "product level health" metrics which you look at when deployed.

I could follow up on this and maybe introduce a split for "trace_metrics" vs "metrics" if people find these too much.

Reply:
The general idea is to use metrics in your dev workflow too.